Xamarin.Forms.Core.UnitTests/ResourceDictionaryTests.cs

```diff
@@ -10,94 +10,100 @@ namespace Xamarin.Forms.Core.UnitTests
 	public class ResourceDictionaryTests : BaseTestFixture
 	{
 		[Test]
-		public void Add ()
+		public void Add()
```

Comment:
@StephaneDelcroix - Apologies on that, I compulsively press Ctrl + K, Ctrl + D. However, the formatting of this class is different to the rest of the code. Can we leave it as is, to bring it in to line with everything else? I will restrain from formatting in the future. :)